File: backend/src/main/java/com/example/bookhub/dto/ReviewDto.java
package com.example.bookhub.dto;

import lombok.Data;

import java.time.LocalDateTime;
import java.util.List;

@Data
public class ReviewDto {

    private Long id;

    private int rating;

    private String content;

    private LocalDateTime createdAt;

    private LocalDateTime updatedAt;

    private String author;

    private List<QuoteDto> quotes;
}

File: frontend/src/app/core/services/review/models/review.dto.ts
import { QuoteDto } from '../../../../shared/models/quote.dto';

export interface ReviewDto {
  readonly id: number;
  readonly rating: number;
  readonly content: string;
  readonly createdAt: Date;
  readonly updatedAt: Date;
  readonly author: string;
  readonly quotes: QuoteDto[];
}
